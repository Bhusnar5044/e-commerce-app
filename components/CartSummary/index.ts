export { CartSummary as default } from './CartSummary';
