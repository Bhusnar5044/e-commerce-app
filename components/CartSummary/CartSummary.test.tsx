import { renderWithStore } from '@test-utils';
import { CartSummary } from './CartSummary';

jest.mock('next/navigation', () => ({
    ...require('next-router-mock'),
    useSearchParams: () => ({ get: jest.fn() }),
}));
describe('CartSummary', () => {
    it('should render properly', () => {
        const { container } = renderWithStore(<CartSummary />);
        expect(container).toMatchSnapshot();
    });
});
