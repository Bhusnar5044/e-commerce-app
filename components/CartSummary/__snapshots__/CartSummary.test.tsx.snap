// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartSummary should render properly 1`] = `
<div>
  <div
    class="mt-16 rounded-lg bg-gray-50 px-4 py-6 sm:p-6 lg:col-span-5 lg:mt-0 lg:p-8"
  >
    <h2
      class="text-lg font-medium text-gray-900"
    >
      Order summary
    </h2>
    <div
      class="mt-6 space-y-4"
    >
      <div
        class="flex items-center justify-between border-t border-gray-200 pt-4"
      >
        <div
          class="text-base font-medium text-gray-900"
        >
          Order total
        </div>
        <div
          class="font-semibold"
        >
          $0.00
        </div>
      </div>
    </div>
    <button
      class="rounded-full bg-black border border-transparent px-5 py-3 disabled:cursor-not-allowed disabled:opacity-50 text-white font-semibold hover:opacity-75 transition opacity-75 cursor-not-allowed w-full mt-6"
      disabled=""
      type="button"
    >
      Checkout
    </button>
  </div>
</div>
`;
